ComponentKitTests/CKComponentGestureActionsTests.mm

```diff
@@ -138,6 +138,16 @@ - (void)testThatWithNoDelegateActionsNoDelegateIsSet
   XCTAssertNil(gesture.delegate, @"Gesture delegate should not be set");
 }
 
+- (void)testThatApplyingANilRecognizerClassResultsInNoRecognizer
+{
+  CKComponentViewAttributeValue attr = CKComponentGestureAttribute(Nil, nullptr, @selector(test));
```

Comment:
My original diff was just adding a CKAssert, then I thought better of it and decided to just gracefully ignore it.